docs/metadata.md

```diff
@@ -37,7 +37,6 @@ metadata:
   applicationType: ANALYTICAL
   externalId: ab12345
   isDevelopmentOnly: false
-  deploymentType: INTERNAL
```

Comment:
Why are you deleting this line?

Reply:
It was a duplicate, `deploymentType` was already mentioned on line 36.